apps/web/src/components/PostEditor/PostEditor.js

```diff
@@ -710,23 +710,31 @@ function PostEditor ({
           )}
         </div>
       )}
-      <div className={cn('PostEditorContent', styles.section, 'flex flex-col !items-start')}>
+      <div className={cn(
+        'PostEditorContent',
+        styles.section,
+        'flex flex-col !items-start border-2 border-transparent',
+        'transition-all duration-200',
+        'focus-within:border-2 focus-within:border-focus'
+      )}>
         {currentPost.details === null || loading
           ? <div className={styles.editor}><Loading /></div>
           : <HyloEditor
-              key={currentPost.id}
               placeholder={isChat ? t('Send a chat to #{{topicName}}', { topicName: currentPost?.topics?.[0]?.name }) : t('Add a description')}
-              onUpdate={handleDetailsChange}
+              onUpdate={(html) => {
```

Comment:
now you are creating an extra function, handleDetailsChange can have this condition in it

Reply:
OK, moved the callback into handleDetailsChange... I also had the ai go through and add comments for what we've worked on and they make sense to me